docs/en/guides/pipeline-integrations/development/sgtk-developer-bootstrapping.md

```diff
@@ -81,8 +81,25 @@ sgtk.LogManager().global_debug = True
 ## Part 3: Authentication
 
 When running a script that uses the Toolkit API outside of an environment where {% include product %} Toolkit has already been started, you will always need to authenticate.
+
 So before you can perform the bootstrapping, you need to authenticate the Toolkit API with your {% include product %} site.
 
+Upon launching Desktop and applications, you will be prompted to sign in to your {% include product %} site (example: `https://mysite.shotgrid.autodesk.com`). 
+
+![sign in](./images/sa-integrations-user-guide-session-launcher-01.png)
```

Comment:
The image is quite large there and the icon is pixelized. We might want to improve that.

http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/5feb147acfb7e23bd6f78537a74b2b0fb83351b4/d587be80/?title=Integrations+User+Guide

Reply:
Thanks @julien-lang ! Let's see how the images look in this next build.